src/actions/__tests__/cursor-actions.test.js

```diff
@@ -0,0 +1,96 @@
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import { moveCursorToNextChunk, updateEditorCursor } from "../actions";
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe("updateEditorCursor - returns correct action", () => {
```

Comment:
let's make adding action tests a requirement for any additions going forward.

Reply:
I think I agree with @wlach's point above but still think that having test coverage is a positive signal. It's not always just about utility.